### PR TITLE
Merging rc2 commits back into master.

### PR DESCRIPTION
**Pull request by av8ramit**
Note: went with master versions of keras, cmake differences, and 1.1 version of configure.

### CODE REVIEW

File: configure [hunk 1]
```diff
@@ -35,6 +35,18 @@ function is_windows() {
   fi
 }
 
+function bazel_clean_and_fetch() {
```
Comment (jhseu) [inline configure]: This function shouldn't be merged back into master, right?

File: tensorflow/tensorflow.bzl [hunk 2]
```diff
@@ -930,6 +930,18 @@ def tf_custom_op_py_library(name,
       deps=deps,)
 
 
+def tf_custom_op_py_library(name, srcs=[], dso=[], kernels=[],
```
Comment (jhseu) [inline tensorflow/tensorflow.bzl]: Duplicated, delete?

**Review by jhseu — CHANGES_REQUESTED** [review]

File: configure (hunk 1 above)
Comment (av8ramit) [inline configure]: I wasn't sure, I can drop it.

Comment (jhseu) [inline configure]: It isn't called, so should be dropped.

Comment (av8ramit) [inline configure]: Done.

File: tensorflow/tensorflow.bzl (hunk 2 above)
Comment (av8ramit) [inline tensorflow/tensorflow.bzl]: Done.

**Review by yifeif — APPROVED** [review]
version changes LGTM

**Review by jhseu — APPROVED** [review]

**Comment by av8ramit** [comment]
Jenkins, test this please.

**Pull request merged**